### PR TITLE
fix: The item of OEM is readonly by permission field

### PR DESCRIPTION
**Pull request by FeiWang1119**
It should be decided if the item of OEM is readonly by the flags [nooverride]

Log: fix the item of OEM is readonly by permission field

### CODE REVIEW

File: dconfig-center/example/configs/example.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
             "name": "I am name",
             "name[zh_CN]": "我是名字",
             "description": "I am description",
-            "permissions": "readwrite",
```
Comment (18202781743) [inline dconfig-center/example/configs/example.json:12]: 单独加个Item，而不是移除掉原来的了，

File: dconfig-center/dde-dconfig-editor/oemdialog.cpp [hunk 2]
```diff
@@ -339,8 +343,8 @@ void OEMDialog::createJsonFile(const QString &fileName, const QList<DStandardIte
 
 QWidget *OEMDialog::getItemWidget(ConfigGetter *getter, DStandardItem *item)
 {
-    const QString &permissions = getter->permissions(item->data(KeyRole).toString());
-    bool canWrite = permissions == "readwrite" ? true : false;
+    const int &flags = getter->flags(item->data(KeyRole).toString());
```
Comment (18202781743) [inline dconfig-center/dde-dconfig-editor/oemdialog.cpp:346]: int就不用引用了把，

File: dconfig-center/example/configs/example.json (hunk 1 above)
Comment (FeiWang1119) [reply dconfig-center/example/configs/example.json:12]: done

File: dconfig-center/dde-dconfig-editor/oemdialog.cpp (hunk 2 above)
Comment (FeiWang1119) [reply dconfig-center/dde-dconfig-editor/oemdialog.cpp]: done

**Review by 18202781743 — APPROVED** [review]

**Pull request merged**